mypy/main.py

```diff
@@ -102,120 +104,152 @@ def process_options(args: List[str]) -> Tuple[List[BuildSource], Options]:
             module to run as script (or None),
             parsed flags)
     """
-    # TODO: Rewrite using argparse.
+
+    footer = textwrap.dedent(
```

Comment:
I don't think it's worth importing textwrap just so you can write this indented. Maybe make it a global constant FOOTER?